### PR TITLE
Features/form validation

### PR DESCRIPTION
**Pull request by Wahaj-Ali**
in this branch
- created a client-side form validation using Javascript
- If the user enters an email even with a single capital letter, a message appears saying to enter the correct email.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -170,4 +170,29 @@ cards.forEach((card) => {
...
+  inputControl.classList.add('.small');
+  inputControl.classList.remove('success');
+};
+
+const validateEmail = () => {
+  const emailformat = /^[a-z0-9_!#$%&'*+=?`{|}~^.-]+@[a-z0-9.-]+$/gm;
+  if (email.value.match(emailformat)) {
+    form.submit();
+  } else {
+    setError(email, 'Please, type your email in lowercase');
```
Comment (franclobo) [inline script.js:191]: - [ ] **[OPTIONAL]** The message should disappear a couple of seconds after showing it, kindly fix it for a better user experience. 👍🏽

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Wahaj-Ali, 

# Required Changes ♻️
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors ✔️ 
- Correct GitHub flow ✔️ 
- Descriptive PR ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] This is a **general requirement**: all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present for a professional repository, please 🙏🏽 add the acknowledgment and LICENSE section. 
- [ ] Kindly add the LICENSE file to your repo and make sure the link on the README file works as expected for a professional repository.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by medaminedev66** [comment]
Hello @Wahaj-Ali and @utsargo 👋,

_This is an additional review as a result of the second opinion request._

Your project is complete. Great work 👏.

[OPTIONAL] To make your Readme file look professional and impressive, it would be excellent if you added the areas the previous reviewer mentioned.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @Wahaj-Ali     :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for meeting all the project requirements good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**